src/components/SimpleSelect.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useContext } from 'react';
+import hyperID from 'hyperid';
+import clsx from 'clsx';
+import Color from 'color';
+import DSRContext from '../contexts/DSRContext';
+
+type OptionType = {
+  value: (string | number),
+  label: (string | number)
+};
+
+type GroupType = {
+  group: string,
+  options: OptionType[]
+};
+
+export type SimpleSelectOptionType = OptionType[] | GroupType[];
+
+export type SimpleSelectProps = {
+  value: (string | number),
+  name: string,
+  className?: string,
+  options: SimpleSelectOptionType,
+  onChange?: (v: (string | number)) => void,
+  required?: boolean,
+  disabled?: boolean,
+  labels?: {
+    label?: string,
+    placeholder?: string
+  }
+};
+
+const defaultLabels = {
+  label: null,
+  placeholder: 'Select an Option',
+};
+
+const SimpleSelect = ({
+  value, onChange = () => {
+  }, className = '',
+  required = false, disabled = false, name, options, labels: _labels,
```

Comment:
we are no longer using variables prefixed with `_`, rename it to something like `initialLabels`